pkg/driverbuilder/docker.go

```diff
@@ -218,13 +228,28 @@ func (bp *DockerBuildProcessor) Start(b *builder.Build) error {
 		}
 	}()
 
-	err = cli.ContainerStart(ctx, cdata.ID, types.ContainerStartOptions{})
+	err = cli.ContainerStart(ctx, cdata.ID, container.StartOptions{})
 	if err != nil {
 		return err
 	}
 
+	runCmd :=
```

Comment:
We create a new cmd that makes required scripts executables and then :
* runs download-libs to download libs repo
* runs download-headers capturing its stdout (see https://github.com/falcosecurity/driverkit/pull/324/files#r1524730462) 
* the stdout from the download-headers step will be used as KERNELDIR env variable for the driverkit.sh script, that is the actual building script